containers/jetty-http/src/main/java/org/glassfish/jersey/jetty/JettyHttpContainer.java

```diff
@@ -82,6 +82,7 @@
     private static final Type RESPONSE_TYPE = (new GenericType<Ref<Response>>() {}).getType();
 
     private static final int INTERNAL_SERVER_ERROR = javax.ws.rs.core.Response.Status.INTERNAL_SERVER_ERROR.getStatusCode();
+    private static final javax.ws.rs.core.Response.Status BAD_REQUEST_STATUS = javax.ws.rs.core.Response.Status.BAD_REQUEST;
```

Comment:
I mean you can move the ``javax.ws.rs.core.Response.Status`` part of the declaration into the ``import javax.ws.rs.core.Response.Status;`` section. Thus the whole block would look like: 

```
    private static final int INTERNAL_SERVER_ERROR = Status.INTERNAL_SERVER_ERROR.getStatusCode();
    private static final Status BAD_REQUEST_STATUS = Status.BAD_REQUEST;
```